### ISSUE TITLE
Superbuild error/rebuild

### ISSUE THREAD

**Issue by paulmelis** [ospray/ospray#359]
I use the following two commands to configure resp. build a superbuild of the devel branch:

```
paulm@llt0022 07:41:~/c/ospray-git/superbuild$ cat .conf.sh 
#!/bin/sh
cmake \
    -DCMAKE_BUILD_TYPE=RelWithDebInfo \
    -DCMAKE_INSTALL_PREFIX=$HOME/software/ospray-superbuild-git \
    -DBUILD_JOBS=5 \
    -DBUILD_EMBREE_FROM_SOURCE=OFF \
    -DBUILD_OIDN=ON \
    -DBUILD_OIDN_FROM_SOURCE=OFF \
    -DINSTALL_IN_SEPARATE_DIRECTORIES=OFF \
    ../scripts/superbuild
paulm@llt0022 07:42:~/c/ospray-git/superbuild$ cat .build.sh 
#!/bin/sh
cmake --build .
```

Currently I see the build error below, which looks like a missing `-I.`. Cleaning out the build dir and rebuilding solves the issue, but I'm wondering about the general workflow with the superbuild and repo updates? For example, I've also noticed that the build command above sometimes does not trigger recompilation of changed files after a git pull. 

```
[ 98%] Building CXX object ospray/CMakeFiles/ospray_module_ispc.dir/volume/amr/AMRAccel.cpp.o
[ 98%] Building CXX object ospray/CMakeFiles/ospray_module_ispc.dir/volume/amr/AMRData.cpp.o
[ 98%] Building CXX object ospray/CMakeFiles/ospray_module_ispc.dir/volume/amr/AMRVolume.cpp.o
[ 99%] Linking CXX shared library ../libospray_module_ispc.so
[ 99%] Built target ospray_module_ispc
Scanning dependencies of target ospray_module_denoiser
[ 99%] Building CXX object modules/denoiser/CMakeFiles/ospray_module_denoiser.dir/DenoiseFrameOp.cpp.o
In file included from /home/paulm/c/ospray-git/modules/denoiser/DenoiseFrameOp.cpp:17:
/home/paulm/c/ospray-git/modules/denoiser/DenoiseFrameOp.h:23:10: fatal error: ospray_module_denoiser_export.h: No such file or directory
   23 | #include "ospray_module_denoiser_export.h"
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
make[5]: *** [modules/denoiser/CMakeFiles/ospray_module_denoiser.dir/build.make:63: modules/denoiser/CMakeFiles/ospray_module_denoiser.dir/DenoiseFrameOp.cpp.o] Error 1
make[4]: *** [CMakeFiles/Makefile2:947: modules/denoiser/CMakeFiles/ospray_module_denoiser.dir/all] Error 2
make[3]: *** [Makefile:152: all] Error 2
make[2]: *** [CMakeFiles/ospray.dir/build.make:112: ospray/stamp/ospray-build] Error 2
make[1]: *** [CMakeFiles/Makefile2:174: CMakeFiles/ospray.dir/all] Error 2
make: *** [Makefile:84: all] Error 2
```

**Comment by paulmelis** [comment]
Oh, rebuilding did not solve the issue, I forgot that one my workstation yesterday I manually added the `-I.` to get it working. So on my laptop this morning I still see the error.

**Comment by paulmelis** [comment]
As I hit upon this again on a different system I opened https://github.com/ospray/ospray/pull/361 to fix. Note: the pull request contains some merges that I shouldn't be there (although only a single file was changed), I need to look into my github steps if I did something weird

**Comment by jeffamstutz** [comment]
I think this has been resolved in `v2.0.0-alpha`. Please let us know if you run into any further issues with the superbuild.
